app/controllers/active_courses_controller.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 class ActiveCoursesController < ApplicationController
-  respond_to :html
+  respond_to :json, :html
 
   def index
```

Comment:
It seems like this is being overloaded for HTML vs JSON, with the same instance variables being set up (and the same queries being run) for both.

Would this make more sense with a different controller action for the html index versus the JSON active courses list?

Reply:
Yeah this is something I changed in [321f966](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/5039/commits/321f966ea4ee2641400b4daaea692d213bffb5c2)

I need the presenter to get the course string prefix. In this case though, I suppose that will always be equal to `Features.default_course_string_prefix`? If so, I could just set it to that.

Reply:
Yeah, for /explore and /active_courses, I think these should always be equal to the default.

For an individual campaign, it could vary based on the campaign's default course type, but that could be done just by loading the campaign itself and doing `campaign.course_string_prefix` instead of going through an otherwise-unnecessary presenter.